smarty/templates/main.tpl

```diff
@@ -206,30 +208,12 @@
         <div id="page-content-wrapper">
             {/if}
             {if $dynamictabs eq "dynamictabs"}
-                {if $console}
```

Comment:
Can it be the case that somebody prefers seeing queries directly on the page, so maybe we need an option to keep 1 or both?